src/scripts/navbar.js

```diff
@@ -0,0 +1,61 @@
+window.onscroll = function () {
+  scrollFunction();
+};
+
+let navbar = document.getElementById("navbar");
+const hamburgerMenu = document.getElementById("menuIcon");
+const overlay = document.querySelector(".overlay");
+const navbarRight = document.getElementById("navbar-right");
+const dropdown = document.querySelectorAll(".dropdown");
+
+function scrollFunction() {
+  if (document.body.scrollTop > 5 || document.documentElement.scrollTop > 5) {
+    navbar.style.height = "58px";
+    document.getElementById("navbar-right").style.lineHeight = "58px";
+    document.getElementById("navbar-right").style.top = "58px";
+    document.getElementById("navbar-left").style.lineHeight = "58px";
+  } else {
+    navbar.style.height = "99px";
+    document.getElementById("navbar-right").style.lineHeight = "99px";
+    document.getElementById("navbar-right").style.top = "135px";
+    document.getElementById("navbar-left").style.lineHeight = "99px";
+  }
+}
+
+for (let i = 0; i < dropdown.length; i++) {
```

Comment:
You could use `foreach` instead of a forloop here

Reply:
Event delegation could also be used, if there are a lot of drop-downs to avoid multiple listeners.